enterprise/app/workflows/workflows.tsx

```diff
@@ -415,6 +418,21 @@ class RepoItem extends React.Component<RepoItemProps, RepoItemState> {
       .executeWorkflow(
         new workflow.ExecuteWorkflowRequest({
           pushedRepoUrl: this.props.repoUrl,
+          // Parse "var=val" pairs from the input field to Record<string, string>
+          env: this.state.runWorkflowActionNames
```

Comment:
Looks like you accidentally swapped the variables - should be runWorkflowEnvs here and actionNames below

Reply:
oh great call. Fixed.